draft-kazuho-httpbis-priority.md

```diff
@@ -509,7 +509,18 @@ as the server did not specify the incremental(`i`) parameter.
 
 # Security Considerations
 
-## Fairness and Coalescing Intermediaries {#fairness}
+## Fairness {#fairness}
+
+As a general guideline, a server SHOULD NOT use the priority values for making
```

Comment:
```suggestion
As a general guideline, a server SHOULD NOT use priority information for making
```

nit: `information` is more correct than `values` because the actual value of a parameter might be ignored. This also makes the terminology consistent in the paragraph and wider doc.